client/src/components/Reservations.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchAllReservations,
+  deleteReservations,
+} from '../redux/ReservationsSlice';
+
+export const Reservations = () => {
```

Comment:
- [ ] When I delete a reservation, It also deletes the house if it finds it:

![image](https://github.com/peterdtitan/BnB-homes/assets/57421823/be455bad-92ec-407f-b1f6-63231179e6e9)



tip: This is because it should delete from the reservations controller, but axios is set to access the homes controller:

![image](https://github.com/peterdtitan/BnB-homes/assets/57421823/6d41c1ba-66a0-46fc-842c-4c02b9357479)

This is also not consistent because it takes a reservation id, and uses that id to find a house, not a reservation.  Therefore, if it finds a house with the same id, it deletes it.

Kindly make it so when I delete a reservation, the house is preserved. 👍

Reply:
- Thanks for pointing this out, it has been resolved. ✅ ✅